src/main/java/gr/grnet/pccapi/endpoint/ProviderEndpoint.java

```diff
@@ -18,6 +25,13 @@ public ProviderEndpoint(ProviderService providerService) {
         this.providerService = providerService;
     }
 
+    @Tag(name="Providers")
+    @APIResponse(
+            responseCode = "200",
+            description = "The list of available providers",
+            content = @Content(mediaType = "application/json",
+                    schema = @Schema(implementation = Provider.class)))
+    @Operation(summary = "Get a list of all available providers")
     @GET
     @Produces(MediaType.APPLICATION_JSON)
     public Response list() {
```

Comment:
add an example as in https://github.com/ARGOeu/pcc-api/pull/11

Reply:
Since we have already described the ProviderResponseDTO.class, the example property is unnecessary. 

You can also use the following content:
`[content = @Content(schema = @Schema(
                    type = SchemaType.ARRAY,
                    implementation = ProviderResponseDTO.class)))](url)`